include/arch/arm/aarch32/exc.h

```diff
@@ -48,14 +48,9 @@
 #endif
 
 #define _EXC_FAULT_PRIO 0
-#ifdef CONFIG_ZERO_LATENCY_IRQS
 #define _EXC_ZERO_LATENCY_IRQS_PRIO 0
-#define _EXC_SVC_PRIO 1
-#define _IRQ_PRIO_OFFSET (_EXCEPTION_RESERVED_PRIO + 1)
-#else
-#define _EXC_SVC_PRIO 0
-#define _IRQ_PRIO_OFFSET (_EXCEPTION_RESERVED_PRIO)
-#endif
+#define _EXC_SVC_PRIO IS_ENABLED(CONFIG_ZERO_LATENCY_IRQS)
```

Comment:
Could these expressions use the ? operator, and make it explicit that we use 0 or 1; would be more readable in my opinion.

Reply:
done